src/hott/07-fibers.rzk.md

```diff
@@ -114,1061 +115,1061 @@ Contractible maps are equivalences:
 We now show that half adjoint equivalences are contractible maps.
 
 ```rzk title="If f is a half adjoint equivalence, its fibers are inhabited"
-#def is-surj-is-half-adjoint-equiv
+#def is-split-surjection-is-half-adjoint-equiv
```

Comment:
In some places we use `hae` and in others (like here) we use `half-adjoint-equiv`. Should we instead stick to one of these everywhere?

Reply:
This PR is supposed to make the code use `half-adjoint-equiv` everywhere, except for in variable names. We can of course also swap the variable name `is-hae-f` with `is-half-adjoint-equiv-f`, although I consider this less important as it is not visible to other parts of the library.

Reply:
I see, we use the shorter version in variables. Makes sense!

Reply:
I see, we use the shorter version in variables. Makes sense!